src/components/Inputs/Input.tsx

```diff
@@ -57,6 +62,7 @@ const Input = ({ placeholder, error, isValid, disabled, onChange, value, ...prop
         className={className}
         disabled={disabled}
         isValid={isValid}
+        onWheel={handleScroll}
```

Comment:
I was originally going to do this, and what made me think this was non-trivial was that we have to consider if *all* `number` inputs should have the scroll wheel behavior disabled / overridden.

IMO it's fine to override all their behavior, it's just something to consider for the future.

Reply:
Yes, for now I also believe that it's a good / safe compromise.